pronto_mia/lib/core/models/deployment_plan.dart

```diff
@@ -14,18 +14,23 @@ class DeploymentPlan {
   String get link => _link;
   final String _link;
 
+  bool get published => _published;
+  final bool _published;
+
   DeploymentPlan(
-    this._id,
-    this._description,
-    this._availableFrom,
-    this._availableUntil,
-    this._link,
-  );
+      this._id,
+      this._description,
+      this._availableFrom,
+      this._availableUntil,
+      this._link,
+      // ignore: avoid_positional_boolean_parameters

```

Comment:
I see 2 options here:
- We disable this linter rule in general
- We refactor all our models, so they take named parameters instead of positional ones

I'm fine with both solutions.